pre_commit/commands/run.py

```diff
@@ -306,6 +306,9 @@ def run(
             f'`git add {config_file}` to fix this.',
         )
         return 1
+    if args.hook_stage in {'prepare-commit-msg', 'commit-msg'} and not args.commit_msg_filename:
+        logger.error('Specify --commit-msg-filename.')
```

Comment:
how about `f'--commit-msg-filename is required for --hook-stage {args.hook_stage}'`

this'll also need a quick test -- can probably follow [this example](https://github.com/pre-commit/pre-commit/blob/1c641b1c28ecc1005f46fdc76db4bbb0f67c82ac/tests/commands/run_test.py#L815-L819)

Reply:
(plus or minus some backticks to make the error message more clear -- hard to nest them on github)

Reply:
sure... didn't realize you were using f-strings in this repo, didn't want to eliminate py3.5 without confirmation

Reply:
Ah nevermind, I was looking at the error message that didn't have them and wasn't paying attention to the other ones.